packages/dx-react-core/src/draggable.tsx

```diff
@@ -79,8 +80,12 @@ export class Draggable extends React.Component<DraggableProps> {
     node.addEventListener('touchstart', this.touchStartListener, { passive: true });
   }
 
+  isRightButton(mouseButton) {
+    return mouseButton === RIGHT_MOUSE_BUTTON;
```

Comment:
This method doesn't use `this`. Should it be method or just utility function?

Reply:
I will add just a function